api/swagger-spec/swagger.yml

```diff
@@ -52,6 +52,26 @@ paths:
           description: unexpected error
           schema:
             $ref: "#/definitions/error"
+  /v3/doctor/{train}/{release}/{uuid}:
```

Comment:
I agree that each deis doctor "report" will be a unique record, and we only need an appropriate unique ID strategy (UUID qualifies). We can build other endpoints that match particular kinds of queries, like say, "all reports sent from a single cluster"; but for submissions, we don't need to worry about scoping them to particular versions or the originating cluster.